docker_pull-from-git.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+docker exec -it "docker_fromthepage-fromthepage-1" sh -c '
+echo "Adding origin..."
+git remote rm origin
+git remote add origin https://github.com/Simon-Dirks/fromthepage
```

Comment:
again, should be a variable.

Reply:
Ah, looks like I included some commits that should not have been in this PR, my bad!